### ISSUE TITLE
Serializable interface deprecated in PHP 8.1

### ISSUE THREAD

**Issue by pklink** [contentful/contentful.php#311]
Hi,  the `Serializable` is marked as deprecated in PHP 8.1 and will be removed in later versions.

I got the following deprecation warning/message:

```
 Contentful\Core\Api\Message implements the Serializable interface, which is deprecated. Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary) in /project-path/vendor/contentful/core/src/Api/Message.php on line 29
```

For further information (w/ a backward compatible "fix") see https://php.watch/versions/8.1/serializable-deprecated.

BR, Pierre

**Comment by pklink** [comment]
PR https://github.com/contentful/contentful-core.php/pull/55

Sorry, I've seen to late, that this problem is located in the `contentful-core` repository

**Comment by Sebb767** [comment]
Released with [v7.0.0](https://github.com/contentful/contentful.php/releases/tag/v7.0.0) :)
